main/inc/ajax/document.ajax.php

```diff
@@ -176,7 +176,7 @@
                 '',
                 '',
                 0,
-                '',
+                'rename',
```

Comment:
Aquí estás cambiando el comportamiento por defecto de Chamilo (warn vs rename) y no entiendo bien porqué.

Reply:
Mmmm OK, I see in BT#18997 that this was a requirement.